exercises/src/test/java/c4_LifecycleHooks.java

```diff
@@ -73,7 +73,10 @@ public void atomic_counter() {
         AtomicInteger counter = new AtomicInteger(0);
 
         Flux<Integer> temperatureFlux = room_temperature_service()
-                .doOnNext(i -> counter.incrementAndGet());
+                .doOnNext(i -> {
+                    System.out.println(i);
```

Comment:
@schananas, I think that this line was added because the description says that one side affect should be also to print out the values. 

https://github.com/schananas/practical-reactor/blob/2c3418db27ee46717c0116e1c10d1d640d8f2de9/exercises/src/test/java/c4_LifecycleHooks.java#L67-L72